Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -484,6 +484,11 @@ extension ActivityStreamPanel: DataObserverDelegate {
             // If there is no pending cache update and highlights are empty. Show the onboarding screen
             self.showHighlightIntro = self.highlights.isEmpty && !self.pendingCacheUpdate
             self.collectionView?.reloadData()
+
+            // Refresh the AS data in the background so we'll have fresh data next time we show.
+            DispatchQueue.global().async {
```

Comment:
runAsync already performs the query in the Background SwiftData thread do we also need to do this here? All we are doing is moving the creation of the query to a BG thread. I also dont want want to expose this to a consumer of PanelDataObserver. This is something that PDO should worry about

Reply:
You're right that this probably doesn't need to go in an `async` call, but I don't see where this would belong in `PanelDataObserver`?